crates/runestick-macros/src/context.rs

```diff
@@ -102,6 +102,7 @@ impl Context {
         let output = FieldAttrs::default();
 
         for attr in attrs {
+            #[allow(clippy::never_loop)] // I guess this is on purpose?
```

Comment:
Yep, meta parsing was removed. But will most likely be added back in again as more things are needed.